etc/db-config.yaml

```diff
@@ -269,6 +269,15 @@
             default_value: false
             public: true
             description: Show canonical compiler and runner version on the team pages.
+        -   name: hide_teams_scoreboard_condition
```

Comment:
Consider phrasing these in a positive form, e.g. show_teams_on_scoreboard: always, after_login, after_submission